<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>stats</class>
 <widget class="QDialog" name="stats">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QTableWidget {
    background-color: #1A2236;
    color: #FFFFFF;
    border: 1px solid #3A3F58;
    gridline-color: #3A3F58;
    font-family: &quot;Inter&quot;;
    font-size: 10pt;
    border-radius: 8px;
    selection-background-color: #0F5FFF;
    selection-color: #FFFFFF;
}

QHeaderView::section {
    background-color: #0F1A2C;
    color: #FFFFFF;
    border: 1px solid #3A3F58;
    padding: 4px;
    font-family: &quot;Montserrat SemiBold&quot;;
    font-size: 10pt;
}

QTableWidget::item:selected {
    background-color: #0F5FFF;
    color: #FFFFFF;
}
QGroupBox {
    border: 1.5px solid #BFC5D2;
    border-radius: 12px;
    margin-top: 25px;
    color: white;
    font-family: &quot;Segoe UI Semibold&quot;;
    font-size: 14pt;
}

QGroupBox::title {
    subcontrol-origin: margin;
    padding: 4px 10px;
}
QPushButton {
    background-color: #0F1A2C;
    color: #FFFFFF;
	border-color: rgb(0, 0, 0);
    border-radius: 10px;
    font-family: &quot;Montserrat SemiBold&quot;;
    font-size: 12pt;
    padding: 6px 12px;
}

QPushButton:hover {
    background-color: #1A2942;
}

QPushButton:pressed {
    background-color: #09111A;
}
QLineEdit {
    background-color: #FFFFFF;
    border: 1px solid #C6D4E1;
    border-radius: 8px;
    padding: 6px;
    color: #0A1320;
    font-family: &quot;Inter&quot;;
    font-size: 11pt;
}

QLineEdit:focus {
    border: 1px solid #00B4FF;
    background-color: #F8FBFF;
}
QComboBox {
    background-color: #FFFFFF;
    border: 1px solid #C6D4E1;
    border-radius: 8px;
    padding: 6px;
    color: #0A1320;
    font-family: &quot;Inter&quot;;
    font-size: 11pt;
}

QComboBox:focus {
    border: 1px solid #00B4FF;
    background-color: #F8FBFF;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;
    border-left: 1px solid #C6D4E1;
}

QComboBox::down-arrow {
    
	image: url(:/new/prefix1/icon.png);
    width: 12px;
    height: 12px;
}
QDateEdit {
    background-color: #FFFFFF;
    border: 1px solid #C6D4E1;
    border-radius: 8px;
    padding: 6px;
    color: #0A1320;
    font-family: &quot;Inter&quot;;
    font-size: 11pt;
}

QDateEdit:focus {
    border: 1px solid #00B4FF;
    background-color: #F8FBFF;
}

QDateEdit::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;
    border-left: 1px solid #C6D4E1;
}

QDateEdit::down-arrow {
    image: url(:/new/prefix1/icon.png);
    width: 12px;
    height: 12px;
}
QDateEdit::up-arrow {
    
	image: url(:/new/prefix1/icon3.webp);
    width: 12px;
    height: 12px;
}

QTabWidget::pane {
    border: 1px solid #3A3F58;
    border-radius: 8px;
    background-color: #1A2236;
}

QTabBar::tab {
    background: #0F1A2C;
    color: #FFFFFF;
    border: 1px solid #3A3F58;
    border-bottom: none;
    border-top-left-radius: 8px;
    border-top-right-radius: 8px;
    padding: 8px 16px;
    font-family: &quot;Montserrat SemiBold&quot;;
    font-size: 10pt;
}

QTabBar::tab:selected {
    background: #0F5FFF;
    color: #FFFFFF;
}

QTabBar::tab:hover {
    background: #1A2942;
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>750</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/new/prefix1/bg.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>40</y>
     <width>701</width>
     <height>691</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 700 14pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Total number of loans :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 700 14pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Number of loans is :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 700 14pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Number of loans is :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 700 14pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Total number of loans is :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 700 14pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Select a Date Range :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 700 14pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Customer with the highest number of loans is :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 700 14pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Select a Type :</string>
    </property>
   </widget>
   <widget class="QComboBox" name="box1">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>102</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="c1">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>102</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm</string>
    </property>
   </widget>
   <widget class="QLabel" name="l2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>150</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;
color: rgb(170, 0, 127);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 700 14pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Select a Status :</string>
    </property>
   </widget>
   <widget class="QComboBox" name="box2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>200</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Active</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Closed</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="c2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>200</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm</string>
    </property>
   </widget>
   <widget class="QLabel" name="l3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>250</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;
color: rgb(170, 0, 127);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QDateEdit" name="db">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>302</y>
      <width>110</width>
      <height>31</height>
     </rect>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>18</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2025</year>
      <month>7</month>
      <day>11</day>
     </datetime>
    </property>
   </widget>
   <widget class="QDateEdit" name="de">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>302</y>
      <width>110</width>
      <height>31</height>
     </rect>
    </property>
    <property name="date">
     <date>
      <year>2026</year>
      <month>7</month>
      <day>12</day>
     </date>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>298</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 700 14pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>to</string>
    </property>
   </widget>
   <widget class="QPushButton" name="c3">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>300</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm</string>
    </property>
   </widget>
   <widget class="QLabel" name="l4">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>350</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;
color: rgb(170, 0, 127);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="l5">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>400</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;
color: rgb(170, 0, 127);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 700 14pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Customer with the highest account balance is :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 700 14pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Customer with the lowest account balance is :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 700 14pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Total number of employees :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>650</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 700 14pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Number of employees per bank branch :</string>
    </property>
   </widget>
   <widget class="QLabel" name="l6">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>450</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;
color: rgb(170, 0, 127);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="l7">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>490</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;
color: rgb(170, 0, 127);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="l8">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>550</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;
color: rgb(170, 0, 127);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="l9">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>650</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;
color: rgb(170, 0, 127);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="l1">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;
color: rgb(170, 0, 127);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>600</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 700 14pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Select Bank Branch :</string>
    </property>
   </widget>
   <widget class="QComboBox" name="box3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>600</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="c4">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>600</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>640</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>20</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #FFFFFF; 
font-family: &quot;Montserrat SemiBold&quot;; 
font-size: 20pt;
font-weight: 700; 
padding: 4px 0;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Statistics&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
